### DISCUSSION TITLE
Page résultat : empreinte environnementale - energie primaire ou eau

### DISCUSSION THREAD

**Discussion by margaux-escande** [cnumr/EcoIndex#146] [Bug]
Dans mes résultats on me parle d'energie primaire avec une unité "I" ? 
Dans le code je vois : data-int = water et historiquement EcoIndex donne GES et Eau. 
Je suppose donc que c'est 18,6 L de consommation d'eau ? 
 
<img width="441" alt="2022-09-19 08_15_10-EcoIndex fr_ Résultat" src="https://user-images.githubusercontent.com/112691869/190959054-3323b105-23c6-4fb5-a527-55825dc235f4.png">

**Comment by tbroyer** [comment]
> Dans mes résultats on me parle d'energie primaire avec une unité "I" ?

C'est un `l` (L minuscule), pas un `I` (i majuscule).
Voir #136 
Voir aussi https://fr.wikipedia.org/wiki/Litre#Symbole
